### DISCUSSION TITLE
Аналог стратегии "--dpi-desync=fake --dpi-desync-any-protocol --dpi-desync-cutoff=n2" с zapret для войса discord

### DISCUSSION THREAD

**Discussion by testing765** [hufrea/byedpi#222] [Q&A]
У меня на запрете работает стратегия `--dpi-desync=fake --dpi-desync-any-protocol --dpi-desync-cutoff=n2`, как ее оформить для byedpi?
Пробую `-d 1 -f 1 -t 3 --udp-fake 1`, дискорд открывается, но войс не работает, почему?

**Comment by SingleInfiniteLoop** [comment]
Try this:
```
-o 1+s -d 3+s -a 2 -t 4
```
You might need to change `-t 4` to `-t 3` if the first one doesn't work.
